Portoflio Desktop Version

TracK92 opened the pull request:
- Use media queries to create responsive web design from the mobile portfolio version.
 - Make adjustable to tablet and desktop screens by using the following media queries:
                      - min-width: 768px
                      - min-width: 992px
                      - min-width: 1200px

sja-thedude commented on index.html:
```diff
@@ -331,17 +447,27 @@ <h4>Skills</h4>
               alt="contact-image"
             />
+            <img
+              class="desktop-bg-image"
+              src="images/contact-form-desktop-background.png"
+              alt="contact-image"
+            />
           </div>
           <div class="contacts-content">
             <div class="contact-intro">
               <h2>Contact me</h2>
-              <p class="pitch">
+              <p class="pitch mobile-pitch">
                 If you have an application you <br />
                 are interested in developing, a <br />
                 feature that you need built or a <br />
                 project that needs coding, I'd <br />
                 love to help with it
               </p>
+              <p class="desktop-pitch">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding, I'd
+                love to help with it
```
Mobile Version:

- [x] Kindly remove the extra spacing according to the Figma design, as shown in the attached loom video.
- [x] Make sure to give spacing in the card below the heading and list icons as per the video attached.
- [x] Kindly implement the button interactions for all the buttons in the project. Interactions like hover and pressed.
<img width="463" alt="Screenshot 2022-06-02 at 6 58 56 PM" src="https://user-images.githubusercontent.com/70269507/171676595-378210f5-a06c-469e-a0e5-03e523bd64ca.png">


Desktop Version:

- [x] Kindly remove the extra spacing as shown in the loom video.
- [x] Kindly make sure that the images are 50% of the card and bigger in size.

trace-kadenyi replied:
[sja-thedude](https://github.com/sja-thedude) thank you for your comments and the video especially. I will make the necessary changes.

sja-thedude reviewed (changes requested):
## Required Changes ♻️

Hi @TracK92, This is @sja-thedude your code reviewer.

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

![768070f3-bc9a-4aba-b8bc-25793e00ea1e_text](https://user-images.githubusercontent.com/70269507/159483175-2f61461d-b352-4cb4-98f8-b173e05ab5ff.gif)

## Highlights
✔️ Professional Readme
✔️ Good PR title and description
✔️ All linters passing

## Required 
- [ ] Kindly watch this loom video [here](https://www.loom.com/share/565aef65ca1a400fa9d89a9f4eab1cb9) in order to make the changes.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

TracK92 commented:
I have made the requested changes for both the mobile and desktop versions. 
   - Removed extra spacing. 
   - Added spacing to the mobile version cards. 
   - Implemented the button interactions. 
   - Increased the image sizes.
  Kindly re-review my work. Thank you.

muneebulrehman commented on index.html:
```diff
@@ -75,253 +83,361 @@ <h3><a href="#">LET'S CONNECT</a></h3>
+                  /></a>
+                </div>
+              </article>
+              <article class="article-two">
+                <h2><a href="#" class="tonic">Tonic</a></h2>
+                <div class="description">
+                  <h5 class="descript"><a href="#">CANOPY</a></h5>
+                  <ul>
+                    <li>
+                      <a href="#"
+                        ><img src="images/circle.svg" alt="circle"
+                      /></a>
+                    </li>
+                    <li><a href="#">Back End Dev</a></li>
+                    <li>
+                      <a href="#"
+                        ><img src="images/circle.svg" alt="circle"
+                      /></a>
+                    </li>
+                    <li><a href="#">2015</a></li>
+                  </ul>
+                </div>
+                <div class="text-sect">
+                  <p class="text-description">
+                    A daily selection of privately personalized reads; no
+                    accounts <br class="breaker" />
+                    or sign-ups required.
+                  </p>
+                </div>
+                <div class="languages">
+                  <ul>
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javaScript</a></li>
+                  </ul>
+                </div>
+                <div>
+                  <button class="see_project">See Project</button>
+                </div>
+              </article>
```
Please make the image size bigger as requested by the previous reviewer. It should be 50% of the card's width. It might have to do something with the dimensions set for this image. 😊
This is how it looks currently.
![image](https://user-images.githubusercontent.com/33268927/171800302-d1e97d08-fa91-41f9-b659-be1a2eabcd92.png)

trace-kadenyi replied:
[muneebulrehman](https://github.com/muneebulrehman) thank you for your observation and feedback. I will adjust the image accordingly.

muneebulrehman reviewed (changes requested):
Hi @TracK92 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good follow-up with the previous reviewer.
- Good PR description
- Good HTML and CSS


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

TracK92 commented:
Dear Reviewer, 
I have adjusted the problematic image accordingly. Kindly re-review my code. Thank you.

KayLemba reviewed (approved):
# STATUS: APPROVED :green_circle: :clap: 

Hello  @TracK92    , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- Nice and Neat User Interface


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

![well-1](https://user-images.githubusercontent.com/64173189/128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c.gif)

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

TracK92 commented:
[KayLemba](https://github.com/KayLemba) thank you for your optional suggestion. I will certainly add it to my README file. 👍
